src/content/docs/infrastructure/prometheus-integrations/view-query-data/translate-promql-queries-nrql.mdx

```diff
@@ -12,10 +12,21 @@ redirects:
   - /docs/integrations/prometheus-integrations/view-query-data/simulate-promql-nrql
 ---
 
+import promqlImgSrc from './images/PROMQL-query-2.png';
+
+import nrqlExampleSrc from './images/NRQL-example-3.png';
```

Comment:
i think these should be named:
- NRQL-query-example-4.png
- NRQL-query-example-5.png

Reply:
There are four images in this doc. We need to import all of them. 

The bigger issue seems to be that I can't seem to add this HTML+React syntax without getting a build error, either locally or on Gatsby cloud. I don't know how to move forward with this one.

Personally, I would prefer to be able to use our standard Markdown image syntax, instead of trying to introduce a new react component-based image syntax for images inside collapsers. From my perspective, this is a bug.

Reply:
i believe the build error was due to wrong image file name is what i was saying

Reply:
![2021-12-15_15-38-19](https://user-images.githubusercontent.com/2952843/146282016-39cae388-5fa3-4f90-a816-5142ea224a61.png)

Reply:
this can be used to work for the time being. long term we can try and get this into a sprint, but it's not clear to me why when MDX is parsed into HTML it doesn't use the right src path. i'm actually curious if using the webpack aliased path fixes this (what i brought up a couple days ago to reduce duplicating image files for same image)

Reply:
Of course... Apparently I've lost my ability to correctly read filenames :D